src/components/WeekDayPicker/index.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import valuePropValidation from './helpers/valuePropValidation';
+import getNormalizedValue from './helpers/getNormalizedValue';
+import { useUniqueIdentifier, useLocale } from '../../libs/hooks';
+import withReduxForm from './../../libs/hocs/withReduxForm';
```

Comment:
instead of use withReduxForm let's use useReduxForm which is a new hook to integrate with redux-form, update with the master branch to get last changes and look at StrongPasswordInput component to see how it is used